Buildings/Experimental/DHC/EnergyTransferStations/Cooling/Examples/CoolingIndirectOpenLoops.mo

```diff
@@ -0,0 +1,203 @@
+within Buildings.Experimental.DHC.EnergyTransferStations.Cooling.Examples;
```

Comment:
The unit test results for model `CoolingIndirectOpenLoops` had a small deviation with the reference results. Consider updating the model or the reference results.
![image](https://user-images.githubusercontent.com/31628742/99756700-f51c1f80-2aaa-11eb-9409-b3deeae222f1.png)

Reply:
Hi @JingWang-CUB, that's strange, I'm not getting the same issue (see below). The initial conditions were not fully specified for this model, which I have since corrected. Please let me know if you are still getting the same issue.

Reply:
Jing updated reference results [here](https://github.com/khinkelman/modelica-buildings/pull/28/commits/fe929b19cb73b4a80a520b5ee51d14bb9599d39e)